packages/appeals-service-api/src/mappers/questionnaire-submission/has-mapper.js

```diff
@@ -0,0 +1,87 @@
+class HasQuestionnaireMapper {
+	mapToPINSDataModel(journeyResponse) {
+		return [
+			{
+				questionnaire: {
+					LPACode: journeyResponse.LPACode,
+					caseReference: journeyResponse.caseReference,
+					isAppealTypeAppropriate: journeyResponse.answers['correct-appeal-type'] ? true : false,
+					doesTheDevelopmentAffectTheSettingOfAListedBuilding: journeyResponse.answers[
+						'affects-listed-building'
+					]
+						? true
+						: false,
+					affectedListedBuildings: () => {
+						if (journeyResponse.answers['affects-listed-building']) {
+							return journeyResponse.answers['listed-building-number'];
+						}
+					},
+					siteWithinGreenBelt: journeyResponse.answers['green-belt'] ? true : false,
+					howYouNotifiedPeople: journeyResponse.answers['notification-method'] ? true : false,
```

Comment:
I think this might need to be an array to support  "howYouNotifiedPeople": [
            "A public notice at the site",
            "Letters to neighbours",
            "Advert in the local press"
        ]